frontend/src/components/ViewParty/ViewParty.js

```diff
@@ -10,6 +10,28 @@ import axios from 'axios';
 
 const ViewParty=(props)=>{
 
+  const joinParty=() =>{
+
+    const headers = {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    };
+  
+  const Link = `${process.env.REACT_APP_URL}/Party/${props.ID}/Update`;
+
+  const payload={
+    Attendees: props.Attendees.push({ID:props.user.ID,Username: props.user.Username})
```

Comment:
I tested this out and it returns an integer. So to fix, declare an array above it, push/concat to it then let the value of the key be the declared array.